docs/source/tensorbay_cli/cli_commands.rst

```diff
@@ -1,3 +1,129 @@
 ##############
  CLI Commands
 ##############
+
+*************
+ gas dataset
+*************
+
+"gas dataset" deals with TensorBay datasets.
+
+Create a dataset.
+
+.. code:: console
+
+   gas dataset [dataset_name]
+
+Note that there is no tb: before the dataset name when creating a dataset.
+
+List all datasets.
+
+.. code:: console
+
+   gas dataset -l
+
+Delete a dataset.
+
+.. code:: console
+
+   gas dataset -d tb:[dataset_name]
+
+
+*********
+ gas ls
+*********
+
+"gas ls" deals with list operation.
+
+List the segments of a dataset.
+
+.. code:: console
+
+    gas ls tb:[dataset_name]
+    gas ls tb:[dataset_name]@[revision]
+    gas ls tb:[dataset_name]#[draft_number]
```

Comment:
needs more detailed explanation for every commands

Reply:
My fault. @linjiX @rexzheng324-c 

Please separate each command again and add explanation to each command, such as:

```rst
List the segments of a specific dataset revision.

.. code:: console

   gas ls tb:[dataset_name]@[revision]
```